src/styles/bloks/card.scss

```diff
@@ -0,0 +1,100 @@
+@use '../utils/variables' as *;
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: $card-width;
+  height: $card-height;
```

Comment:
Ensure that the variables '$card-width' and '$card-height' are defined in the 'variables' file. If they are not, you need to define them or import them from the correct file.